web_ir_actions_act_window_message/static/src/js/web_ir_actions_act_window_message.js

```diff
@@ -25,8 +25,19 @@ odoo.define('web.web_ir_actions_act_window_message', function(require)
                     text: action.close_button_title || _t('Close'),
                     click: function() {
                         // refresh the view before closing the dialog
-                        self.inner_widget.active_view
-                            .controller.recursive_reload();
+                        var controller = self.inner_widget.active_view.controller;
```

Comment:
that's what I meant with refactoring: Move this to its own function, something like `ir_actions_act_window_message_refresh` (to be sure we won't overwrite some action manager function) and do this there.